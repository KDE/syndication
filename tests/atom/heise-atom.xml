<?xml version="1.0" encoding="utf-8"?>
<!-- uses only <updated> but not <published> -->

<feed xmlns="http://www.w3.org/2005/Atom">
  <title>heise online News</title>
  <subtitle>Nachrichten nicht nur aus der Welt der Computer</subtitle>
  <link href="http://www.heise.de/newsticker/" />
  <link rel="self" href="http://www.heise.de/newsticker/heise-atom.xml" />
  <updated>2009-01-11T00:12:07+01:00</updated>
  <author>
    <name>heise online</name>
  </author>
  <id>http://www.heise.de/newsticker/</id>

  <entry>
    <title>Was war. Was wird.</title>
    <link href="http://www.heise.de/newsticker/Was-war-Was-wird--/meldung/121495/from/atom10" />
    <id>http://www.heise.de/newsticker/meldung/121495</id>
    <updated>2009-01-11T00:12:07+01:00</updated>
    <summary>Was war. Was wird.</summary>
  </entry>

  <entry>
    <title>Satyam-Chef festgenommen</title>
    <link href="http://www.heise.de/newsticker/Satyam-Chef-festgenommen--/meldung/121490/from/atom10" />
    <id>http://www.heise.de/newsticker/meldung/121490</id>
    <updated>2009-01-10T18:01:54+01:00</updated>
    <summary>Satyam-Chef festgenommen</summary>
  </entry>

  <entry>
    <title>25 Prozent mehr Ger채te mit WLAN</title>
    <link href="http://www.heise.de/newsticker/25-Prozent-mehr-Geraete-mit-WLAN--/meldung/121488/from/atom10" />
    <id>http://www.heise.de/newsticker/meldung/121488</id>
    <updated>2009-01-10T16:38:07+01:00</updated>
    <summary>25 Prozent mehr Ger채te mit WLAN</summary>
  </entry>

  <entry>
    <title>Google k체ndigt Mac- und Linux-Versionen von Chrome an</title>
    <link href="http://www.heise.de/newsticker/Google-kuendigt-Mac-und-Linux-Versionen-von-Chrome-an--/meldung/121487/from/atom10" />
    <id>http://www.heise.de/newsticker/meldung/121487</id>
    <updated>2009-01-10T15:29:44+01:00</updated>
    <summary>Google k체ndigt Mac- und Linux-Versionen von Chrome an</summary>
  </entry>

  <entry>
    <title>Operas Wii-Browser soll Acid3-Test bestehen</title>
    <link href="http://www.heise.de/newsticker/Operas-Wii-Browser-soll-Acid3-Test-bestehen--/meldung/121485/from/atom10" />
    <id>http://www.heise.de/newsticker/meldung/121485</id>
    <updated>2009-01-10T20:32:27+01:00</updated>
    <summary>Operas Wii-Browser soll Acid3-Test bestehen</summary>
  </entry>
</feed>

