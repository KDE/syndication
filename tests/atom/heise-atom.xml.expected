# Feed begin ######################
title: #heise online News#
link: #http://www.heise.de/newsticker/#
description: #Nachrichten nicht nur aus der Welt der Computer#
# Person begin ####################
name: #heise online#
# Person end ######################
# Item begin ######################
id: #http://www.heise.de/newsticker/meldung/121495#
title: #Was war. Was wird.#
link: #http://www.heise.de/newsticker/Was-war-Was-wird--/meldung/121495/from/atom10#
description: #Was war. Was wird.#
datePublished: #Sun Jan 11 00:12:07 2009#
dateUpdated: #Sun Jan 11 00:12:07 2009#
# Item end ########################
# Item begin ######################
id: #http://www.heise.de/newsticker/meldung/121490#
title: #Satyam-Chef festgenommen#
link: #http://www.heise.de/newsticker/Satyam-Chef-festgenommen--/meldung/121490/from/atom10#
description: #Satyam-Chef festgenommen#
datePublished: #Sat Jan 10 18:01:54 2009#
dateUpdated: #Sat Jan 10 18:01:54 2009#
# Item end ########################
# Item begin ######################
id: #http://www.heise.de/newsticker/meldung/121488#
title: #25 Prozent mehr Ger채te mit WLAN#
link: #http://www.heise.de/newsticker/25-Prozent-mehr-Geraete-mit-WLAN--/meldung/121488/from/atom10#
description: #25 Prozent mehr Ger채te mit WLAN#
datePublished: #Sat Jan 10 16:38:07 2009#
dateUpdated: #Sat Jan 10 16:38:07 2009#
# Item end ########################
# Item begin ######################
id: #http://www.heise.de/newsticker/meldung/121487#
title: #Google k체ndigt Mac- und Linux-Versionen von Chrome an#
link: #http://www.heise.de/newsticker/Google-kuendigt-Mac-und-Linux-Versionen-von-Chrome-an--/meldung/121487/from/atom10#
description: #Google k체ndigt Mac- und Linux-Versionen von Chrome an#
datePublished: #Sat Jan 10 15:29:44 2009#
dateUpdated: #Sat Jan 10 15:29:44 2009#
# Item end ########################
# Item begin ######################
id: #http://www.heise.de/newsticker/meldung/121485#
title: #Operas Wii-Browser soll Acid3-Test bestehen#
link: #http://www.heise.de/newsticker/Operas-Wii-Browser-soll-Acid3-Test-bestehen--/meldung/121485/from/atom10#
description: #Operas Wii-Browser soll Acid3-Test bestehen#
datePublished: #Sat Jan 10 20:32:27 2009#
dateUpdated: #Sat Jan 10 20:32:27 2009#
# Item end ########################
# Feed end ########################

